latex/ledger-spec.tex

```diff
@@ -331,13 +332,10 @@ \section{Serialization}
 assumed to hold true independently of the storage medium and style of data
 organization chosen for an implementation.
 
-\clearpage
```

Comment:
did you mean to remove these `\clearpage`s?

Reply:
Oops